### ISSUE TITLE
ccm ocasionally fails due to lack of internet on github actions

### ISSUE THREAD

**Issue by adespawn** [scylladb/scylla-ccm#628]
When used as part of integration tests, run on github actions workers, scylla-ccm occasionally (once every ~20 launches) fails due to lack of internet:

```
File "/usr/lib/python3.10/socket.py", line 833, in create_connection

    sock.connect(sa)
OSError: [Errno 101] Network is unreachable
```

When failing action is re-run, everything works normally (network issue no longer appears).

Full error message from GH actions for the failing step: [error.log](https://github.com/user-attachments/files/18059999/error.log)
Link to the repository: [repo - link](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/)
Link to specific GH action launch: [log - link](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/runs/12225798378/attempts/1?pr=95)
Other failed action log: [log2 - link](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/runs/12225894301/job/34100431287)
Link to used workflow: [workflow - link](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/miscellaneous-types/.github/workflows/integration-tests.yml)

**Comment by fruch** [comment]
@adespawn I would guess you got it figured out, since it doesn't sound like issue with ccm itself

**Comment by adespawn** [comment]
Hard to say. The problem was appearing when using ccm on GH actions, but I didn't see it for a long time, so it's likely some changes made by GitHub solved this issue.

**Comment by fruch** [comment]
> Hard to say. The problem was appearing when using ccm on GH actions, but I didn't see it for a long time, so it's likely some changes made by GitHub solved this issue.

yes, sounds like a environment networking issue, closing this for now
